docs/_data/prague-2020-sessions.yaml

```diff
@@ -1,8 +1,9 @@
 - title: The Importance of Using Analytics and Feedback for your Documentation
   slug: the-importance-of-using-analytics-and-feedback-for-your-documentation-karissa-van-baulen
   series: Write the Docs Prague
-  series_slug: the-importance-of-using-analytics-and-feedback-for-your-documentation-karissa-van-baulen
```

Comment:
These were just wrong originally, right?

Reply:
Yes. I fixed the script for this, but that was after we created this file.